libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionNoSetupTest.kt

```diff
@@ -244,3 +240,64 @@ private fun buildTripSession(
         eHorizonSubscriptionManager = mockk(relaxed = true),
     )
 }
+
+class StatusWithVoiceInstructionUpdateUtil {
```

Comment:
You can make this `object`, then you won't need to use `companion object`. 
Also it looks like it can just be a private function here.